infrastructure-playbooks/purge-docker-cluster.yml

```diff
@@ -325,6 +318,12 @@
       name: ceph-facts
       private: false
 
+  - name: gather monitors facts
```

Comment:
Don't we need this before the ceph-facts role?
It was the initial issue AFAIR.

Reply:
correct